.travis.yml

```diff
@@ -63,7 +64,7 @@ install:
  - if [[ $BUILD_WHEELS != 1 ]]; then pip install -r tests/requirements.txt ; fi
  - if [[ $MK_DOCS == y ]]; then pip install -r docs/requirements.txt; fi
  # Install librdkafka and confluent_kafka[avro] if not building wheels
- - if [[ $BUILD_WHEELS != 1 ]]; then pip install -U protobuf && tools/bootstrap-librdkafka.sh --require-ssl ${LIBRDKAFKA_VERSION} tmp-build ; fi
+ - if [[ $BUILD_WHEELS != 1 ]]; then pip install -U protobuf && tools/bootstrap-librdkafka.sh --require-ssl ${LIBRDKAFKA_SRC_CODE_VERSION} tmp-build ; fi
```

Comment:
I think this is a good permanent change which makes sure that the Python client is always buildable with the latest master of librdkafka (which should always be buildable as well). This will give us early error detection rather than later when bumping the released version.